File: src/main/java/sharev/gathering/repository/GatheringRepository.java
package sharev.gathering.repository;

import java.util.List;
import java.util.UUID;
import org.springframework.data.jpa.repository.JpaRepository;
import sharev.gathering.entity.Gathering;
import sharev.team.entity.Team;

public interface GatheringRepository extends JpaRepository<Gathering, UUID> {
    List<Gathering> findAllByTeam(Team team);
}

File: src/main/kotlin/sharev/gathering/adapter/outbound/jpa/repository/GatheringRepository.kt
package sharev.gathering.adapter.outbound.jpa.repository

import org.springframework.data.jpa.repository.JpaRepository
import sharev.gathering.adapter.outbound.jpa.entity.GatheringJpaEntity
import sharev.team.adapter.outbound.jpa.entity.TeamJpaEntity
import java.util.*

interface GatheringRepository : JpaRepository<GatheringJpaEntity, UUID> {
    fun findAllByTeam(team: TeamJpaEntity): List<GatheringJpaEntity>
}
